jenkins/debian_c.sh

```diff
@@ -18,11 +18,18 @@ build_folder=$build_root"/cmake/debian_linux"
 # Set the default cores
 CORES=$(grep -c ^processor /proc/cpuinfo 2>/dev/null || sysctl -n hw.ncpu)
 
+if [ $CORES -eq 1 ]
+then
+    LOADLIMIT=$CORES
+else
+    LOADLIMIT=$(perl -e "print int($CORES * 0.8)")
```

Comment:
>LOADLIMIT [](start = 4, length = 9)

Please print this value and CORES value for reference. We will need it for debugging on Azure DevOps hosted VMs which have variance in I/O and cores. We have no control on that.